tests/test_clickhouse.py

```diff
@@ -10,17 +13,20 @@
 @TestScenario
 @Name("test_ch_001. Insert quorum")
 def test_ch_001(self):
-    require_zookeeper()
-
-    create_and_check(
+    util.require_zookeeper()
+    chit_data = manifest.get_chit_data(util.get_full_path("templates/tpl-clickhouse-19.11.yaml"))
+    kubectl.launch(f"delete chit {chit_data['metadata']['name']}", ns=settings.test_namespace)
+    kubectl.create_and_check(
         "configs/test-ch-001-insert-quorum.yaml",
         {
-            "apply_templates": {"templates/tpl-clickhouse-19.11.yaml"},
+            "apply_templates": {"templates/tpl-clickhouse-20.8.yaml"},
             "pod_count": 2,
             "do_not_delete": 1,
         })
+    chi = manifest.get_chi_name(util.get_full_path("configs/test-ch-001-insert-quorum.yaml"))
+    chi_data = kubectl.get("chi", ns=settings.test_namespace, name=chi)
+    util.wait_clickhouse_cluster_ready(chi_data)
```

Comment:
look like latest version of clickhouse-operator doesn't update `<remote_servers>` section even when ClickhouseInstallation CR have `Complete` status